play/PlayerRules/CharacterCreation/03_Feats.txt

```diff
@@ -466,62 +452,101 @@ Proficient: Explosives
 Active Feats
 ====================
 
-Adrenaline Rush [20* X]: 
-Prerequisites:
-Run double your movement distance for X # of turns.
-
-Aimed Shot [10 Nanites]: 
+Adrenaline Rush [4*X Stamina]: 
 Prerequisites:
-Add an action to your normal Shooting time. During this time you concentrate 
-on your target in order to deal crit damage on your next shot. This is akin to 
-seeing a chink in armour and aiming for it, or shooting a target''s arm so 
-they cannot use their weapon, etc.
-
-Bunker:
+During a move action, +1 movement speed per X (up to a max of +10 movement),
+for that action only. This feat can only be used once per turn.
+Adrenaline Rush can be taken again to reduce the stamina cost to 2*X.
+
+Careful Shot [8 Stamina]: 
+Prerequisites: PER 6
+Use your stamina during an aim action to help you more carefully aim at the target. 
+Automatically critical hit on your next shot, if you hit. This is akin to seeing 
+a chink in armour and aiming for it, or shooting a target's arm so they cannot use their
+weapon, etc.
+
+Bunker [25 Stamina]:
 Prerequisites: High-Technology class (Monk, Paladin, Adept, Priest) or 
-Knowledge [Nanites] > 40
-Expend 25 Nanites, gain 25% to a piece of cover for 4 turns
-
-Combat Roll [40 Nanites]:
-Prerequisites: Dex of 5 or more
-Move 6m in one action, no more, no less. Enemies attacking you take a +20% 
-to miss when attacking you for the turn after you use this feat, and you take a 
-+15% to miss for the action after using this feat ends. If this malus would 
-not run out until after this turn, it's effect wraps around to the beginning of 
-your next turn. Cannot be used in concert with run n gun.
-
-Cover Slide [20 Nanites]:
-Prerequisites: Dex of 4 or more
+	Knowledge [Nanites] > 40
+Add +3 miss chance to a piece of cover for 4 turns
+
+Combat Roll [30 Stamina]:

```

Comment:
Add that the miss chance applies to reflex rolls made by enemies too, example being pinned by a sniper or LMG